app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,9 @@
+class LikesController < ApplicationController
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @like = Like.new(user: @user, post: @post)
+    @like.save
```

Comment:
- [ ] It is always good practice to check if the resource has been saved eg:
```ruby
  if @like.save
          format.html { redirect_to @post }
        else
          format.html { redirect_to @post }
        end
```
The reason behind this is, Incase something happens or there was an error your app will not break